app/src/main/java/com/sunasterisk/boringweather/ui/current/CurrentFragment.kt

```diff
@@ -161,9 +229,19 @@ class CurrentFragment : BaseFragment(), CurrentContract.View,
                 ?.let(this::showDailyWeather)
             showTodaySummaryWeather(todaySummary)
             showForecastSummaryWeather(forecastSummary)
+            finishRefresh()
         }, animTime)
     }
 
+    override fun showError(errorStringRes: Int) {
+        val errorMsg = getString(errorStringRes)
+        requireContext().showToast(errorMsg, Toast.LENGTH_LONG)
+        finishRefresh()
+    }
+
+    private fun ifCurrentWeatherIsOutdated(currentWeather: CurrentWeather) =
+        currentWeather.copy(city = City.default) == CurrentWeather.default
```

Comment:
```suggestion
    private fun isCurrentWeatherIsOutdated(currentWeather: CurrentWeather) =
        currentWeather.copy(city = City.default) == CurrentWeather.default
```
hàm trả về true false thế nào nên là is not if